### PR TITLE
MOSIP-25864 MOSIP-26010 MOSIP-25751 Added attribute Name in personalized card template.

### PR DESCRIPTION
**Pull request by kameshsr**

### CODE REVIEW

File: resident/resident-service/src/main/java/io/mosip/resident/util/TemplateUtil.java [hunk 1]
```diff
@@ -111,34 +121,74 @@ public Tuple2<Map<String, String>, ResidentTransactionEntity> getCommonTemplateV
         }
         return Tuples.of(templateVariables, residentTransactionEntity);
     }
+    private String getAttributeValue(String attributes, String languageCode) {
```
Comment (loganathan-sekaran) [inline resident/resident-service/src/main/java/io/mosip/resident/util/TemplateUtil.java]: Please add comment with description of what this method does.

File: resident/resident-service/src/main/java/io/mosip/resident/util/TemplateUtil.java [hunk 2]
```diff
@@ -240,13 +290,25 @@ public Tuple2<Map<String, String>, String> getAckTemplateVariablesForDownloadPer
...
-        templateVariables.put(TemplateVariablesConstants.PURPOSE, getPurposeFromResidentTransactionEntityLangCode(
-        		residentTransactionEntity, languageCode));
+        templateVariables.put(TemplateVariablesConstants.PURPOSE, addAttributeInPurpose(getPurposeFromResidentTransactionEntityLangCode(
+        		residentTransactionEntity, languageCode), getEntityFromEventId(eventId).getPurpose(), languageCode));
         return Tuples.of(templateVariables, Objects.requireNonNull(
                 this.env.getProperty(ResidentConstants.ACK_DOWNLOAD_PERSONALIZED_CARD_TEMPLATE_PROPERTY)));
     }
 
-     public  Tuple2<Map<String, String>, String> getAckTemplateVariablesForOrderPhysicalCard(String eventId, String languageCode, Integer timeZoneOffset) {
+    private String addAttributeInPurpose(String fileText, String purpose,
```
Comment (loganathan-sekaran) [inline resident/resident-service/src/main/java/io/mosip/resident/util/TemplateUtil.java:316]: Add description about the method.

**Review by loganathan-sekaran — CHANGES_REQUESTED** [review]

File: resident/resident-service/src/main/java/io/mosip/resident/util/TemplateUtil.java (hunk 1 above)
Comment (kameshsr) [reply resident/resident-service/src/main/java/io/mosip/resident/util/TemplateUtil.java]: done.

File: resident/resident-service/src/main/java/io/mosip/resident/util/TemplateUtil.java (hunk 2 above)
Comment (kameshsr) [reply resident/resident-service/src/main/java/io/mosip/resident/util/TemplateUtil.java:316]: Done.

**Review by loganathan-sekaran — APPROVED** [review]

**Pull request merged**